ipfsapi/encoding.py

```diff
@@ -1,3 +1,4 @@
+# -*- encoding: utf-8 -*-
```

Comment:
It is only required for Python 2 and only if you want to use any non-ASCII characters in your script files (including comments, strings, …). Python 3 assumes utf-8 by default.